README.md

```diff
@@ -386,7 +386,7 @@ track_event('page_view', source_domain: parsed[:root_domain]) if parsed
 def internal_link?(url, base_domain)
   return false unless DomainExtractor.valid?(url)
 
-  DomainExtractor.parse(url)[:root_domain] == base_domain
+  DomainExtractor.parse(url).root_domain == base_domain
```

Comment:
Since we are showing an example of using `[:…]` in the previous example, we might as well show an example of a method call here.